packages/babel-parser/src/plugins/flow/index.ts

```diff
@@ -3146,7 +3146,12 @@ export default (superClass: {
         }
       }
 
-      return super.checkParams(node, allowDuplicates, isArrowFunction);
+      return super.checkParams(
+        node,
+        allowDuplicates,
+        isArrowFunction,
+        strictModeChanged,
+      );
```

Comment:
here interestingly function argument was missing comparing to overridden method signature